src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static int getRandomBall() {
+            Random random = new Random();
```

Comment:
Creating a new instance of 'Random' every time the method is called is not efficient. You should create a single instance of 'Random' at the class level and reuse it.